app/models/course/leaderboard_settings.rb

```diff
@@ -35,13 +38,45 @@ def display_user_count
     @settings.display_user_count || 30
   end
 
-  # Returns the number of users to be displayed on the leaderboard
+  # Set the number of users to be displayed on the leaderboard
   #
   # @param [Integer] count The number of users to be displayed
   def display_user_count=(count)
     @settings.display_user_count = count
   end
 
+  # Returns whether group leaderboard is enabled (disabled by default).
+  #
+  # @return [Boolean] Setting on whether group leaderboard is enabled.
+  def enable_group_leaderboard
+    @group_leaderboard_settings.settings(:enabled) == true
+  end
+
+  # Enable or disable the option to display group leaderboard
+  #
+  # @param [Boolean|Fixnum|String] option Setting on whether group leaderboard is enabled.
+  #   By default, simple_form provides '0' and '1' for boolean fields.
+  #   This method will handle this conversion to Boolean.
+  def enable_group_leaderboard=(option)
+    option = TRUE_VALUES.include?(option)
```

Comment:
Did some digging, didn't find it in ActiveModel, but the closest I got was [this stackoverflow answer](http://stackoverflow.com/questions/22372593/how-are-values-cast-when-assigning-to-a-boolean-field-in-rails-activerecord). From there, it was casted (somewhere in `ActiveRecord::ConnectionAdapters::Column` based on these values:

```ruby
TRUE_VALUES = [true, 1, '1', 't', 'T', 'true', 'TRUE', 'on', 'ON'].to_set
FALSE_VALUES = [false, 0, '0', 'f', 'F', 'false', 'FALSE', 'off', 'OFF'].to_set
```